docs/datamodel/colltypes.rst

```diff
@@ -7,28 +7,40 @@ Collection Types
 *Collection types* are special generic types used to group homogeneous or
 heterogeneous data.
 
+.. eql:type:: std::array
 
-.. eql:type:: std::tuple
+    :index: array
 
-    :index: tuple
+    Arrays represent a one-dimensional homogeneous ordered list.
 
-    A tuple type is a heterogeneous sequence of other types.
+    Array indexing starts at zero.
 
-    Tuple elements can optionally have names,
-    in which case the tuple is called a *named tuple*.
+    With the exception of other array types, any type can be used as an
+    array element type.
 
-    A tuple type can be explicitly declared in an expression or schema
-    declaration using the following syntax:
+    An array type is created implicitly when an
+    :ref:`array constructor <ref_eql_expr_array_ctor>` is
+    used:
 
-    .. eql:synopsis::
+    .. code-block:: edgeql-repl
 
-        tuple "<" <element-type>, [<element-type>, ...] ">"
+        db> SELECT [1, 2];
+        {[1, 2]}
 
-    A named tuple:
+        db> SELECT (INTROSPECT TYPEOF [1, 2]).name;
```

Comment:
doesn't work right now :( drop it

Reply:
Removed the broken `INTROSPECT` from the docs.